src/main/flight/mixer_tricopter.c

```diff
@@ -536,12 +537,26 @@ STATIC_UNIT_TESTED void tailTuneModeThrustTorque(thrustTorque_t *pTT, const bool
         }
         else
         {
+            if (IsDelayElapsed_ms(pTT->lastAdjTime_ms, 1000))
```

Comment:
1
Think I agree with purpose but not clear about solution yet. I guess we still want a 250 ms timer for the sticks AND for the gyro, so we don't make bad samples when servo angle is bad but the yaw is not up to speed yet.
2
Don't see the point. A small tailTuneGyroLimit must be good for accuracy so it shouldn't grow quicker than necessary.